corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -521,3 +521,17 @@ def get_wrapped_ledger_values(domain, case_ids, section_id, entry_ids=None):
         query = query.entry(entry_ids)
 
     return [StockLedgerValueWrapper.wrap(row) for row in query.run().hits]
+
+
+def get_aggregated_ledger_values(domain, case_ids, section_id, entry_ids=None):
+    # todo: figure out why this causes circular import
```

Comment:
i'm going to keep them until i'm done with this work and then can remove or hopefully fix it